app/controllers/home_controller.rb

```diff
@@ -86,6 +86,12 @@ def sign_up
       return
     end
 
+    begin
+      UserMailer.account_request_reply(home_params[:email]).deliver_now
+    rescue
```

Comment:
could you throw in the err.message as well? or at least Rails.logger that one

Reply:
Did backtrace instead.